README.md

```diff
@@ -184,9 +184,10 @@ $ aws ecs update-service --service sinopia-homepage --region us-west-2 --cluster
 ```
 
 ## Release Management
-The steps to create a tagged release of the Sinopia's Linked Data Editor are as follows:
+The steps to create a major tagged release of the Sinopia's Linked Data Editor are as follows:
```

Comment:
Wouldn't we want to follow these steps for minor and patch releases as well?

Reply:
Yes, I agree! Having the same steps for major, minor, and patch makes it less confusing. Remove *major* from latest commit.